website/templates/log_list.mako

```diff
@@ -55,9 +55,19 @@
                                     </span>
                                 </span>
                                 <!-- Log actions are the same as their template name -->
-                                <span data-bind="template: {name: log.action, data: log}"></span>
```

Comment:
I've reverted the change in this template because it caused all logs to be the same ("changed a non-bibliographic contributor..."). I'll have QA retest and see if the original issue is still addressed.